src/components/preview_panel/PreviewIframe.tsx

```diff
@@ -433,6 +446,7 @@ export const PreviewIframe = ({ loading }: { loading: boolean }) => {
                     ? "Deactivate component selector"
                     : "Select component"}
                 </p>
+                <p>{isMac ? "Meta + Shift + C" : "Ctrl + Shift + C"}</p>
```

Comment:
let's use ⌘ for Meta and ⇧ for Shift (this is more idiomatic and what Chrome DevTools does)